modules/meshrenderinggl/CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+#--------------------------------------------------------------------
+# Inviwo fancymeshrenderer Module
+ivw_module(MeshRenderingGL)
+
+#--------------------------------------------------------------------
+# Add header files
+set(HEADER_FILES
+    include/modules/meshrenderinggl/datastructures/halfedges.h
+    include/modules/meshrenderinggl/algorithm/calcnormals.h
```

Comment:
Since the calc normals algorithm and processor is not using OpenGL and that this module requires OpenGL 4.2 I would suggest that they be moved to another module

Reply:
the modules does not require opengl4.2 only some features of the fancy renderer